### PR TITLE
Create eip155-4689.json and eip155-4690.json

### PR DESCRIPTION
**Pull request by raullenchai**

### CODE REVIEW

**Comment by raullenchai** [comment]
@ligi - pls help with the code review.

File: _data/chains/eip155-4690.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "IoTeX Network Mainnet",
```
Comment (ququzone) [inline _data/chains/eip155-4690.json:2]: should be `IoTeX Network Testnet` ?

**Review by ligi — CHANGES_REQUESTED** [review]
short names must be unique and shorter than name

**Comment by raullenchai** [comment]
> short names must be unique and shorter than name

Fixed. PTAL

**Comment by ligi** [comment]
Oh sorry - missed the notification - just saw it when you closed the issue - would merge - why did you close?

**Comment by raullenchai** [comment]
> Oh sorry - missed the notification - just saw it when you closed the issue - would merge - why did you close?

Reopened, I thought I need another pr to pass the tests

**Review by ligi — APPROVED** [review]

**Comment by ligi** [comment]
no you passed the tests - was just human (me) error here - sorry for the delay

**Pull request merged**